src/lib/fetch-content-api-file-string.js

```diff
@@ -0,0 +1,11 @@
+const fetchFileString = require('./fetch-file-string')
+
+const API_URL = `https://mktg-content-api-hashicorp.vercel.app`
+const API_ASSETS = `/api/assets`
+async function fetchContentApiFileString({ product, filePath, version }) {
+  const [p, v, fp] = [product, version, filePath].map(encodeURIComponent)
+  const url = `${API_URL}${API_ASSETS}?product=${p}&version=${v}&asset=${fp}`
+  return await fetchFileString(url)
+}
+
+module.exports = fetchContentApiFileString
```

Comment:
I've pulled in `fetch-content-api-file-string` and `fetch-file-string` from `main`, for use in fetch the [version.json file from hashicorp/vagrant](https://github.com/hashicorp/vagrant/blob/main/website/data/version.json).